### ISSUE TITLE
Images with spaces in their path are not rendered correctly

### ISSUE THREAD

**Issue by xhacker** [CocoaPods/cocoadocs.org#376]
See the second image in http://cocoadocs.org/docsets/TagListView/0.2/, and the corresponding GitHub page https://github.com/xhacker/TagListView.

I'm guessing something is wrong here https://github.com/CocoaPods/cocoadocs.org/blob/master/classes/readme_manipulator.rb#L44, but I'm not really familiar with ruby.

**Comment by orta** [comment]
Interesting, thanks for the report

**Comment by segiddins** [comment]
Yup, the issue there is that the html already has the space encoded as `%20`, so it gets re-encoded to `%2520`, which is wrong

**Comment by xhacker** [comment]
Bump.

**Comment by xhacker** [comment]
This is not stale. The issue still exists.
